src/NHibernate/Type/GenericOrderedSetType.cs

```diff
@@ -11,6 +11,9 @@ namespace NHibernate.Type
 	[Serializable]
 	public class GenericOrderedSetType<T> : GenericSetType<T>
 	{
+		// ReSharper disable once StaticMemberInGenericType
```

Comment:
I think this warning is useful, but in this case, I want to have exact this side effect - having a distinct static field per closed generic type.